publisher-sdk/src/androidTest/java/com/criteo/publisher/privacy/ConsentDataTest.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.privacy
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.preference.PreferenceManager
+import com.criteo.publisher.mock.MockedDependenciesRule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import javax.inject.Inject
+
+class ConsentDataTest {
+  @Rule
+  @JvmField
+  val mockedDependenciesRule = MockedDependenciesRule()
+
+  @Inject
+  private lateinit var context: Context
+
+  private lateinit var sharedPreferences: SharedPreferences
+
+  @Before
+  fun setUp() {
+    sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
+  }
+
+  @After
+  fun tearDown() {
+    sharedPreferences.edit().clear().commit()
+  }
+
+  @Test
```

Comment:
Would you mind adding a test to check the behavior when the shared pref is empty ?